v2/pkg/cli/executor.go

```diff
@@ -44,22 +44,59 @@ import (
 var (
 	mirrorlongDesc = templates.LongDesc(
 		` 
-		Create and publish user-configured mirrors with a declarative configuration input.
-		used for authenticating to the registries. 
+		Mirror OCP Release, operator catalog and additional images using a declarative configuration file as an input. 
 
-		The podman location for credentials is also supported as a secondary location.
+		The command line uses a declarative configuration file as the input to discover from where to get the container images and copy them to the destination specified in the command line. 
```

Comment:
```suggestion
		The command line uses a declarative configuration file as the input to discover where to get the container images and copy them to the destination specified in the command line. 
```

Reply:
Done :white_check_mark:

Thanks for the suggestion.